src/chat/chat_panel.rs

```diff
@@ -252,161 +253,166 @@ live_design! {
     }
 
     ChatPanel = {{ChatPanel}} {
-        width: Fill,
-        height: Fill,
-        margin: {top: 0, left: 20, right: 20, bottom: 20},
-
-        flow: Overlay,
-
-        no_downloaded_model = <View> {
-            width: Fill,
-            height: Fill,
+        flow: Overlay
+        width: Fill
+        height: Fill
 
-            flow: Down,
-            align: {x: 0.5, y: 0.5},
+        <View> {
+            flow: Overlay
+            width: Fill
+            height: Fill
+            padding: {left: 25, right: 25, bottom: 20},
```

Comment:
Added 5 units to the right add some little extra space for the button. Added 5 units to the left just for symmetry.